src/resources/Pipelines/StatementGroups/StatementGroupsInterfaces.ts

```diff
@@ -7,17 +7,47 @@ import {
 
 export interface StatementGroupList {
     groups: StatementGroupModel[];
+
+    /**
+     * The total number of matching statements across all pages of results.
+     */
     totalCount: number;
+
     groupComposition: PipelineGroupsComposition;
 }
 
 export type StatementGroupModel = PermanentStatementGroup | CampaignStatementGroup;
 
 interface StatementGroupModelBase {
+    /**
+     * The unique identifier of the statement group.
+     */
     id: string;
+
+    /**
+     * The name of the statement groups.
```

Comment:
```suggestion
     * The name of the statement group.
```

Unless it is for multiple groups? 🤔